lib/wasp_vm.ex

```diff
@@ -160,19 +160,19 @@ defmodule WaspVM do
     {reply, vm} =
       case func_addr do
         :not_found -> {{:error, :no_exported_function, fname}, vm}
-        addr -> execute_func(vm, addr, args, opts[:gas_limit])
+        addr -> execute_func(vm, addr, args, [opts[:gas_limit], opts[:trace]])
```

Comment:
Lets just pass the entire opts here since at this point that's all we're using it for. We can split it out later if we need to.

Reply:
Fixed